packages/terra-clinical-detail-view/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
```

Comment:
This should be `Added` for all the changelogs.

Reply:
Done